var/spack/repos/builtin/packages/xsdk-examples/package.py

```diff
@@ -12,28 +12,57 @@ class XsdkExamples(CMakePackage):
 
     homepage = 'http://xsdk.info'
     url      = 'https://github.com/xsdk-project/xsdk-examples/archive/v0.1.0.tar.gz'
+    git      = "https://github.com/xsdk-project/xsdk-examples"
 
     maintainers = ['acfisher', 'balay', 'balos1', 'luszczek']
 
+    version('develop', branch='master')
```

Comment:
We discourage naming a `master` branch `develop` even when there is no `develop` branch as that can be confusing to users who are used to having the two as separate branches in their workflows.

```suggestion
    version('master', branch='master')
```

Reply:
I will "waive" this suggestion for now but do recommend it be changed in a subsequent PR.